### PR TITLE
Add ability to use custom labels in SpecFlow plugin

### PR DESCRIPTION
**Pull request by evgeniy-skt**
### Context
Can't use Layer, AllureId fields with SpecFlow Plugin

#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

**Review by neparij — CHANGES_REQUESTED** [review]

**Comment by neparij** [comment]
Hello guys! We are very glad to see your contributions.

Sounds funny because I pushed almost the same changes recently (https://github.com/allure-framework/allure-csharp/commit/f5ac293d084e5479fa0fdc7fe3354059d0795a3a) :)

Your PR looks really nice, just need to make some changes on review.
I will be glad to see you in the list of contributors!

🍕 ❤️

File: Allure.Features/allureConfig.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     },
     "labels": {
       "owner": "^owner:?(.+)",
-      "severity": "^(normal|blocker|critical|minor|trivial)"
+      "severity": "^(normal|blocker|critical|minor|trivial)",
+      "label": "(.+)\\((.+)\\)"
```
[the remark this answers is not in the document]
Comment (neparij) [reply Allure.Features/allureConfig.json:35]: Also, you can use internal property in PluginConfiguration this way:
```csharp
internal string label => "^label:([\\w]+):(.+)";
```

File: Allure.Net.Commons/Model/allure2.Extensions.cs [hunk 2]
```diff
@@ -129,6 +129,15 @@ public static Label Host(string value)
                 value = value
             };
         }
+        
+        public static Label CustomLabel(string key, string value)
```
[the remark this answers is not in the document]
Comment (alekskulakov) [reply Allure.Net.Commons/Model/allure2.Extensions.cs]: fixed

File: Allure.Features/allureConfig.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (alekskulakov) [reply Allure.Features/allureConfig.json]: Fixed regex.

[the remark this answers is not in the document]
Comment (alekskulakov) [reply Allure.Features/allureConfig.json]: And I'm not sure about 
> internal string label => "^label:([\\w]+):(.+)";

Do we really need this? In this case we can't customize a regex pattern in the config file.

**Review by neparij — APPROVED** [review]

**Pull request merged**